content/guides/identity-management/quick-start/4-logging-into-app.md

```diff
@@ -0,0 +1,440 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Logging into the app with Okta
+
+With the Okta, Box, and basic application set up, we can turn our attention to
+the first step in the application code flow, the Okta login.
+
+During the Okta login we will employ the OpenID Connect (OIDC) frameworks of
+the language used to redirect the user to Okta to log in and pass Okta user
+information back to the application. Those Okta user details will in turn be
+used to validate and create Box users in the next step.
+
+This section will walk you through:
+
+* Setting up the application configuration skeleton.
+* Defining the routes for the chosen framework to handle user traffic.
+* Passing Okta user information to the next Box user validation step.
+
+## Set up the Skeleton
+
+<Choice option='programming.platform' value='node' color='none'>
+In your local application directory, load the `server.js` file created in
+step 1.
+
+Start by copying the following package definitions and configuration
+information into the file.
+
+```js
```

Comment:
This is a large code sample. Could be split up into seperate copy-pastes with some text inbetween to explain each but. Not sure